packages/cojson/src/sync.ts

```diff
@@ -306,8 +306,8 @@ export class SyncManager {
       }
 
       // Fill the missing known states with empty known states
-      if (!peer.optimisticKnownStates.has(coValue.id)) {
-        peer.setOptimisticKnownState(coValue.id, "empty");
+      if (!peer.getKnownState(coValue.id)) {
+        peer.setKnownState(coValue.id, "empty");
```

Comment:
Before we were checking optimistic known state because there could have been the case were we had it but not the knownState.

Now it's not possible anymore, so switched to `getKnownState` to make the intent more clear.